# Goal orientation for robot
float32 robot_orientation

# Commands for VISCA camera pant-tilt-zoom
int32 cam_pan
int32 cam_tilt
uint32 cam_panspeed
uint32 cam_tiltspeed

# Path to soundfile to be played back
string sound_toplay
